apps/prairielearn/src/schemas/schemas/infoAssessment.json

```diff
@@ -160,6 +160,11 @@
     "module": {
       "description": "Module that this assessment belongs to, as defined in infoCourse.json.",
       "type": "string"
+    },
+    "sharedSourcePublicly": {
```

Comment:
sharedSourcePublicly should be changed to shareSourcePublicly

shared_source_publicly should be share_source_publicly

Reply:
Renamed